odpf/optimus/runtime_service.proto

```diff
@@ -293,6 +301,17 @@ message JobStatus {
     google.protobuf.Timestamp scheduled_at = 2;
 }
 
+message JobEvent {
+    enum Type {
+        INVALID = 0;
+        SLA_MISS = 1;
+        FAILURE = 2;
+        SUCCESS = 3;
+    }
+    Type type = 1;
+    map<string, string> values = 2;
```

Comment:
what all we be passed in the values?

Reply:
Details related to the event like airflow log url